### PR TITLE
Fix  #2486

### LINKED ISSUE #2486: blender verification error,  IndexError: list index out of range
#### Encountered on:

Windows
GOLEM Version: 0.13.0+dev200.g004d33d '
Protocol Version: 26-testnet '
golem_messages Version: 1.17.2 '

from `golem.log`
```
2018-03-23 14:23:56 ERROR    golem.task.taskthread               b'Task computing error'
Traceback (most recent call last):
  File "C:\Users\ederenn\Projects\golem\golem\docker\task_thread.py", line 106, in run
    self.task_computer.task_computed(self)
  File "C:\Users\ederenn\Projects\golem\golem\task\localcomputer.py", line 94, in task_computed
    self.computation_success(task_thread)
  File "C:\Users\ederenn\Projects\golem\golem\task\localcomputer.py", line 105, in computation_success
    self.success_callback(task_thread.result, self._get_time_spent())
  File "C:\Users\ederenn\Projects\golem\apps\blender\task\blendercropper.py", line 213, in crop_rendered
    verification_context.errback, self.crop_counter)
  File "C:\Users\ederenn\Projects\golem\apps\blender\task\blendercropper.py", line 167, in _render_one_crop
    minx, maxx, miny, maxy = verify_ctx.crop_values[crop_number]
IndexError: list index out of range
```

### PR DESCRIPTION
**Pull request by Elfoniok**

### CODE REVIEW

File: apps/blender/task/blendercropper.py [hunk 1]
```diff
@@ -188,30 +186,32 @@ def generate_ctd(subtask_info, script_src):
...
-    def crop_rendered(self, results, time_spend, verification_context):
-        self.rendered_crops_results[self.crop_counter] \
+    def crop_rendered(self, results, time_spend, verification_context,
+                      crop_number):
+        self.rendered_crops_results[crop_number] \
             = [results, time_spend, verification_context]
-        if self.crop_counter == 3:
-            self.crop_rendering_finished()
+        crop_number += 1
+        if crop_number == 3 or crop_number == 6:
```
Comment (badb) [inline apps/blender/task/blendercropper.py]: Can we have some consts here instead of magic numbers?

Comment (Elfoniok) [reply apps/blender/task/blendercropper.py]: sure :)

File: apps/blender/task/blendercropper.py [hunk 2]
```diff
@@ -188,30 +188,36 @@ def generate_ctd(subtask_info, script_src):
...
 
-    def crop_rendering_finished(self):
-        for i in range(1, 4):
+    def crop_rendering_finished(self, begin, end):
+        for i in range(begin, end):
             self.rendered_crops_results[i][2].success(
                 self.rendered_crops_results[i][0],
                 self.rendered_crops_results[i][1],
-                self.rendered_crops_results[i][2], i-1)
+                self.rendered_crops_results[i][2], i-self.crop_counter)
```
Comment (mfranciszkiewicz) [inline apps/blender/task/blendercropper.py:204]: Isn't `self.crop_counter` always `0`?

Comment (Elfoniok) [reply apps/blender/task/blendercropper.py:204]: No i set it after first stage to 3.

Comment (mfranciszkiewicz) [reply apps/blender/task/blendercropper.py:204]: Ack, thanks.

**Review by mfranciszkiewicz — APPROVED** [review]

**Pull request merged**